internal/server/volume/server.go

```diff
@@ -137,7 +139,24 @@ func (s *Server) FormatVolume(context context.Context, request *internal.FormatV
 		klog.Errorf("failed FormatVolume %v", err)
 		return response, err
 	}
+	return response, nil
+}
+
+func (s *Server) WriteVolumeCache(context context.Context, request *internal.WriteVolumeCacheRequest, version apiversion.Version) (*internal.WriteVolumeCacheResponse, error) {
+	klog.V(4).Infof("calling WriteVolumeCache with request: %+v", request)
+	response := &internal.WriteVolumeCacheResponse{}
+
+	filePath := request.FilePath
```

Comment:
This needs to be. Since the type is `string`, the compiler did not catch it. 
```
	volumeID := request.VolumeId
	if volumeID == "" {
		klog.Errorf("volume id empty")
		return response, fmt.Errorf("volume id empty")
	}
	err := s.hostAPI. WriteVolumeCache(volumeID)
```

Reply:
thanks for catching it. updated. PTAL